cpp/src/example_libxtreemfs/example_libxtreemfs.cpp

```diff
@@ -46,7 +46,7 @@ int main(int argc, char* argv[]) {
     // Create a new instance of a client using the DIR service at
     // 'demo.xtreemfs.org' (default port 32638).
     client = xtreemfs::Client::CreateClient(
-        "demo.xtreemfs.org:32638",
```

Comment:
Revert this! We may update the demo server for being able to use quotas after this is merged to the master branch.